client/components/search/index.jsx

```diff
@@ -89,6 +90,7 @@ const Search = React.createClass( {
 			isOpen: false,
 			dir: undefined,
 			fitsContainer: false,
+			highlightOnFocus: true,
```

Comment:
I wonder if `disableHighlight` would be better semantically for an optional prop? Fits the same pattern as `disableAutocorrect`.